### PR TITLE
Fixed styling for the Eventlist filter

### PR DESCRIPTION
**Pull request by itsisak**
aligned checkboxes with text and added padding to heading so that the menu isnt on the border-bottom line
Skjermbilde 2022-10-12 kl  19 44 52 #2946

### CODE REVIEW

File: app/routes/events/components/EventList.css [hunk 1]
```diff
@@ -60,3 +61,8 @@
     margin-bottom: 40px;
   }
 }
+
+.filterButtons label {
+  display: flex;
+  align-items: center;
+}
```
Comment (ivarnakken) [inline app/routes/events/components/EventList.css:68]: Please include a line at the end of the file 😅 

Make the change, then amend it to your commit:
`git add <file>`
`git commit --amend --no-fix`

**Review by ivarnakken — APPROVED** [review]
Awesome 💯

**Review by Arashfa0301 — APPROVED** [review]
LGTM

File: app/routes/events/components/EventList.css (hunk 1 above)
Comment (Arashfa0301) [reply app/routes/events/components/EventList.css]: Or`git add .` to add all the files 😀

Comment (itsisak) [reply app/routes/events/components/EventList.css]: fixa det, gjorde det bare i vscode source control

Comment (Arashfa0301) [reply app/routes/events/components/EventList.css]: Faster anyways that writing Eventlist.css. len(".") < len("Eventlist.css")

**Pull request merged**